src/services/ExpressionBuilder.ts

```diff
@@ -41,14 +41,27 @@ interface Options {
    * Sets if the values are JSON encoded
    */
   decodeFilters: boolean;
+
+  /**
+   * Keys to ignore
+   */
+  ignoreKeys?: string[];
+
+  /**
+   * Filter type
+   */
+  type: 'indexed' | 'field';
```

Comment:
If you change `type`  to `filterType`, both comments would be not needed. The rest of the code should be slightly more verbose too.

Reply:
Changed in https://github.com/grafana/logs-drilldown/pull/1074/commits/9c114c7293e92ec6c3101d3e9db392330d97fba9